etc/example-config.yaml

```diff
@@ -33,6 +33,8 @@ openshift:
   bearer_token_file: ""
   image_pull_policy: IfNotPresent
   sandbox_role: "edit"
+  keep_namespace: false
+  keep_namespace_on_error: false
```

Comment:
the deployment above shows `keep_namespace_on_error = true`, but in the example we default it to false.

Reply:
the default above needs to be false